cou/apps/auxiliary.py

```diff
@@ -160,7 +160,9 @@ def _get_change_require_osd_release_step(self) -> PreUpgradeStep:
         """
         ceph_mon_unit, *_ = self.units
         return PreUpgradeStep(
-            description="Ensure require-osd-release option matches with ceph-osd version",
+            description=(
+                "Ensure that the 'require-osd-release' option matches the 'ceph-osd' version"
+            ),
```

Comment:
not-blocking: I saw this is part of dev/data-plane, when I was creating PR for COUApplication and COUUnit (not ready), but using bracket here is not necessary. You are doing it in multiple places.

Reply:
Since it's parameter, you do not need to use brackets and split it into multiple lines.